project-base/storefront/components/Pages/CategoryDetail/CategoryBestsellers/CategoryBestsellers.tsx

```diff
@@ -0,0 +1,63 @@
+import { Button } from 'components/Forms/Button/Button';
+import { ListedProductFragmentApi } from 'graphql/generated';
+import { useGtmSliderProductListViewEvent } from 'gtm/hooks/productList/useGtmSliderProductListViewEvent';
+import { GtmProductListNameType } from 'gtm/types/enums';
+import { useComponentUpdate } from 'hooks/helpers/useComponentUpdate';
+import { useTypedTranslationFunction } from 'hooks/typescript/useTypedTranslationFunction';
+import { useRouter } from 'next/router';
+import { useState } from 'react';
+import { CategoryBestsellersListItem } from './CategoryBestsellersListItem';
+
+const NUMBER_OF_VISIBLE_ITEMS = 3;
+
+type CategoryBestsellersProps = {
+    products: ListedProductFragmentApi[];
+};
+
+export const CategoryBestsellers: FC<CategoryBestsellersProps> = ({ products }) => {
+    const t = useTypedTranslationFunction();
+    const router = useRouter();
+    const [isCollapsed, setIsCollapsed] = useState(false);
+
+    useComponentUpdate(() => setIsCollapsed(false), [router.asPath]);
```

Comment:
This doesn't seem to work at any scenarios which I've tested. If you need to persist this state, let's remove this and save value in Zustand.

Reply:
Tbh I took some of the things pretty brainlessly from a project and this is the effect of it :laughing: The point of this line of code was to hide the products every time you go from a category to category. Therefore, the point is not to remember, but correctly forget the state. However, after some consideration and testing, I realized it's not needed at all, so I just removed it :slightly_frowning_face: